onnxruntime/test/providers/cpu/tensor/gather_op_test.cc

```diff
@@ -322,7 +322,9 @@ TEST(GatherOpTest, Gather_axis1_neg_indices2d_int8) {
                          {1, 0, 2, 1,
                           11, 10, 12, 11,
                           21, 20, 22, 21});
-  test.Run(OpTester::ExpectResult::kExpectSuccess, "", {kTensorrtExecutionProvider});  //TensorRT: Assertion `regionRanges != nullptr' failed
+  // OpenVINO EP: Disabled temporarily
```

Comment:
more details on the failure we're excluding?

Reply:
Added details